applications/asset_tracker_v2/tests/location_module/prj.conf

```diff
@@ -1,9 +1,11 @@
 #
-# Copyright (c) 2021 Nordic Semiconductor ASA
+# Copyright (c) 2022 Nordic Semiconductor ASA
```

Comment:
```suggestion
# Copyright (c) 2021 Nordic Semiconductor ASA
```

Copyright year should stay first year of publication of the file.

Reply:
Github sees this as renaming although it's deleting gnss module tests and adding location module tests so I'd say this should be 2022.

Reply:
@coderbyheart Please check this and resolve if the response is fine or comment why it's not.